ohsome/clients.py

```diff
@@ -740,10 +740,30 @@ def count(self):
 class _OhsomeClientContributionsAggregated(_OhsomePostClient):
     @property
     def density(self):
-        return _OhsomePostClient(
+        return _OhsomeClientContributionsAggregatedDensity(
```

Comment:
please move these changes to the respective branch, then rebase on that branch. It is not a good idea to mix changes of different scope in one MR.

see #134